docs/standard/generics/math.md

```diff
@@ -0,0 +1,199 @@
+| <xref:System.Numerics.IFloatingPoint%601>              | Exposes APIs common to floating-point types.                                                           |
+| <xref:System.Numerics.IFloatingPointIeee754%601>       | Exposes APIs common to floating-point types that implement the IEEE 754 standard.                      |
+| <xref:System.Numerics.INumber%601>                     | Exposes APIs common to comparable number types (effectively the "real" number domain).                 |
+| <xref:System.Numerics.INumberBase%601>                 | Exposes APIs common to all number types (effectively the "complex" number domain).                     |
+| <xref:System.Numerics.ISignedNumber%601>               | Exposes APIs common to all signed number types (such as the concept of `NegativeOne`).                 |
+| <xref:System.Numerics.IUnsignedNumber%601>             | Exposes APIs common to all unsigned number types.                                                      |
+| <xref:System.Numerics.IAdditiveIdentity%602>           | Exposes the concept of `(x + T.AdditiveIdentity) == x`.                                                |
+| <xref:System.Numerics.IMinMaxValue%601>                | Exposes the concept of `T.MinValue` and `T.MaxValue`.                                                  |
+| <xref:System.Numerics.IMultiplicativeIdentity%602>     | Exposes the concept of `(x * T.MultiplicativeIdentity) == x`.                                          |
+
+<sup>1</sup>The binary floating-point types are <xref:System.Double> (`double`), <xref:System.Half>, and <xref:System.Single> (`float`).
+
+<sup>2</sup>The binary integer types are <xref:System.Byte> (`byte`), <xref:System.Int16> (`short`), <xref:System.Int32> (`int`), <xref:System.Int64> (`long`), <xref:System.IntPtr> (`nint`), <xref:System.SByte> (`sbyte`), <xref:System.UInt16> (`ushort`), <xref:System.UInt32> (`uint`), <xref:System.UInt64> (`ulong`), and <xref:System.UIntPtr> (`nuint`).
+
+The interface you're most likely to use directly is <xref:System.Numerics.INumber%601>, which roughly corresponds to a "real" number. If a type implements this interface, it means that a value has a sign and well-defined order, and you can compare it to other values of the same type. <xref:System.Numerics.INumberBase%601> confers more advanced concepts, such as "complex" and "imaginary" numbers, for example, the square root of a negative number. Other interfaces, such as <xref:System.Numerics.IFloatingPointIeee754%601>, were created because not all operations make sense for all number&mdash;for example, calculating the floor of a number only makes sense for floating-point types. In the .NET base class library, the floating-point type <xref:System.Double> implements <xref:System.Numerics.IFloatingPointIeee754%601> but <xref:System.Int32> does not.
```

Comment:
nit: This reads awkward to me (number -> number type):

```suggestion
The interface you're most likely to use directly is <xref:System.Numerics.INumber%601>, which roughly corresponds to a "real" number. If a type implements this interface, it means that a value has a sign and well-defined order, and you can compare it to other values of the same type. <xref:System.Numerics.INumberBase%601> confers more advanced concepts, such as "complex" and "imaginary" numbers, for example, the square root of a negative number. Other interfaces, such as <xref:System.Numerics.IFloatingPointIeee754%601>, were created because not all operations make sense for all number types&mdash;for example, calculating the floor of a number only makes sense for floating-point types. In the .NET base class library, the floating-point type <xref:System.Double> implements <xref:System.Numerics.IFloatingPointIeee754%601> but <xref:System.Int32> does not.
```